scene/scene-layer-selection/README.md

```diff
@@ -1,25 +1,35 @@
-# Scene Layer Selection
+# Scene layer selection
 
-Select clicked features in a scene.
+Identify features in a scene to select.
 
-![](SceneLayerSelection.png)
+![Image of scene layer selection](SceneLayerSelection.png)
+
+## Use case
+
+You can select features to visually distinguish them with a selection color or highlighting. This can be useful to demonstrate the physical extent or associated attributes of a feature, or to initiate another action such as centering that feature in the scene view.
 
 ## How to use the sample
 
-Click on a building in the scene layer to select it. Unselect buildings by clicking away from the buildings.
+Click a building in the scene layer to select it. Deselect buildings by tapping away from the buildings.
 
 ## How it works
 
-To select geoelements in a scene layer:
-
 1. Create an `ArcGISSceneLayer` passing in the URL to a scene layer service.
-2. Use `sceneView.setOnMouseClicked` to get the screen click location `Point2D`.
-3. Call `sceneView.identifyLayersAsync(sceneLayer, point2D, tolerance, false, 1)` to identify features in the scene.
-4. From the resulting `IdentifyLayerResult`, get the list of identified `GeoElement`s with `result.getElements()`.
+2. Use `sceneView.setOnTouchListener(...)` to get the screen tap location `screenPoint`.
+3. Call `sceneView.identifyLayersAsync(sceneLayer, screenPoint, tolerance, false, 1)` to identify features in the scene.
```

Comment:
```suggestion
3. Call `sceneView.identifyLayersAsync(sceneLayer, point2D, tolerance, false, 1)` to identify features in the scene.
```

Reply:
Good catch, this must have been wrong for a while!